apps/backend/src/eventHandlers/logging.ts

```diff
@@ -177,6 +182,58 @@ export default function createHandler() {
             event.questionarystep.questionaryId.toString()
           );
           break;
+        case Event.INSTRUMENTS_REMOVED_FROM_TECHNIQUE:
+          {
+            let description = '';
+            if (event.boolean && event.inputArgs) {
+              const obj = JSON.parse(event.inputArgs);
+              const instruments =
+                await instrumentDataSource.getInstrumentsByIds(
+                  obj[0].instrumentIds
+                );
+
+              const technique = await techniqueDataSource.getTechnique(
+                obj[0].techniqueId
+              );
+
+              description = `Selected instruments: ${instruments?.map((instrument) => instrument.name).join(', ')} is removed from technique: ${technique?.name}`;
+            }
+
+            await eventLogsDataSource.set(
+              event.loggedInUserId,
+              event.type,
+              json,
+              '',
+              description
+            );
+          }
+          break;
+        case Event.INSTRUMENTS_ASSIGNED_TO_TECHNIQUE:
+          {
+            let description = '';
+            if (event.boolean && event.inputArgs) {
+              const obj = JSON.parse(event.inputArgs);
+              const instruments =
+                await instrumentDataSource.getInstrumentsByIds(
+                  obj[0].instrumentIds
+                );
+
+              const technique = await techniqueDataSource.getTechnique(
+                obj[0].techniqueId
+              );
+
+              description = `Selected instruments: ${instruments?.map((instrument) => instrument.name).join(', ')} is attached to technique: ${technique?.name}`;
+            }
+
+            await eventLogsDataSource.set(
+              event.loggedInUserId,
+              event.type,
+              json,
+              '',
+              description
+            );
+          }
```

Comment:
Can we include the changedObjectId as the techniqueId rather than empty string same as above.